_posts/2016-12-02-november.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: "Dev Diary (November 2016)"
+author: "Corey Montella"
+tags: []
+---
+
+_(Eve is a new programming language, and this is our development blog. If you’re new to Eve, [start here](http://play.witheve.com))_
+
+With the launch of Eve v0.2 at the end of October, the early weeks of November were spent reacting and responding to feedback from new users to the ever-growing Eve community.
+
+### What's new in Eve Alpha v0.2.2?
+
+For a full list of the changes, check out the [release notes][1.1]. We've got several much-requested features in there for you. Let's take a look at those in detail.
+
+#### Eve npm package
+
+We published a package for Eve on npm, so you can now install Eve like so:
+
+```
+npm install -g witheve
+``` 
+
+This will give you a global Eve installation that you can invoke with the command `eve` from anywhere. Doing so will start an Eve server, in the same way `npm start` currently does.
+
+#### Workspaces
+
+We're introducing the ability to run Eve in a custom project workspace. To create a new Eve project workspace, just create a folder with an empty file named `package.json`. Now, start Eve from within this folder. Eve recognizes that it's starting in an Eve project, and will serve *.eve files from within this directory instead of the Eve examples folder. 
+
+Furthermore, you can serve various assets for your Eve programs. Eve currently looks for the following folders in your project, and will serve files from them:
+
+- `/css` - for serving stylesheets
```

Comment:
let's just mention assets, the fact that it serves from /css and /fonts right now is a hack because our repo isn't very well organized. All of that should be in /assets for us too.